src/main/java/uk/gov/hmcts/reform/coh/controller/validators/Validation.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.coh.controller.validators;
+
```

Comment:
This is called the Strategy Pattern. Instead of having all those Enum validation routines copy pasted code to loop through each validation, we'll get this class to do it for us.